tests/run-pass/hashmap.rs

```diff
@@ -22,4 +23,17 @@ fn main() {
     assert_eq!(map.values().fold(0, |x, y| x+y), num*(num-1)/2);
 
     // TODO: Test Entry API, Iterators, ...
+
+}
+
+fn main() {
+    let _map : HashMap<i32, i32, BuildHasherDefault<collections::hash_map::DefaultHasher>> = Default::default();
+
+    // TODO: Implement random number generation on OS X
+    if cfg!(not(target_os = "macos")) {
+        let map_normal: HashMap<i32, i32> = HashMap::new();
+        test_map(map_normal);
+    } else {
+        test_map(_map);
```

Comment:
Can you move the `let _map` into this conditional branch to avoid the unused variable on other platforms? Then you can also avoid the `_` in the name.

Reply:
TBH I don't see the point, this code just looks weird.  Why test *creation* on both but not the rest of the test?

The purpose of this is mostly to test Miri, not to test the HashMap. For that, HashBrown actually runs its test suite in Miri.  I don't see a good reason to use multiple different hash functions here.

Reply:
My intention was to ensure that Miri's test coverage didn't regress (e.g. ensure that we catch any regressions with creating a HashMap with a custom hasher).

I'll clean it up.